docs/accessories/developer/itemstack_components.md

```diff
@@ -0,0 +1,284 @@
+|`"invalid_slots"`| An array of string values of invalid slots that such stack **can not** be equipped into when attempted |
+
+### Stack Size
+
+To adjust the data-driven components ability to stack, you can use this component to either use the stacks max size or use a overridden size instead but such size needs to be less than the stack size or that will be used instead.
+
+```JSON
+{
+    "accessories:stack_size": {
+        "use_stack_size": false,
+        "size_override": 69
+    }
+}
+```
+
+| <div style="width:118px">Field</div> | Usage |
+|--|--|
+|`"use_stack_size"`| Use the stack's given max size as the amount allowed to stack to within Accessories |
+|`"size_override"`| The amount that the size should be overridden to instead of stack size |
+
+### Accessory Nest
+
+Accessories has a system to allow for nesting an Accessory within another Accessory which can only be used within code or by creating a ItemStack with the given Nest component. 
+
+```JSON
+{
+    "accessories:nested_accessories": {
+        "accessories": [
+            {
+                "id": "minecraft:diamond",
+                "count": 1,
+                "components": {
+                    "accessories:attributes": {
+                        "modifiers": [
+                            {
+                                "type": "minecraft:attack_damage",
+                                "id": "example:increased_damage_boost",
+                                "amount": 1,
+                                "operation": "add_value",
+                                "slot_name": "any",
+                                "is_stackable": false,
+                            }
+                        ]
+                    }
+                }
+            }
+            {
+                "id": "minecraft:emerald",
+                "count": 1,
+                "components": {
+                    "accessories:attributes": {
+                        "modifiers": [
+                            {
+                                "type": "minecraft:attack_speed",
+                                "id": "example:increased_damage_speed",
+                                "amount": 1,
+                                "operation": "add_value",
+                                "slot_name": "any",
+                                "is_stackable": false,
+                            }
+                        ]
+                    }
+                }
+            }
+        ]
+    }
+}
+```
+
+| <div style="width:118px">Field</div> | Usage |
+|--|--|
+|`"accessories"`| A list of `ItemStack`s that have various components and will be used to get Accessory info from when equipping the nest |
+
+!!! warning "Nest Equitability"
+    You will either need to add the Slot Validation component to the nest or add the given item that the nest is made from to the given tag for the slot using the Tag Predicate for equipping
```

Comment:
Well just because it is an accessory nest, it does not mean it can just be equipped so the warning is just telling people to add the ability to equip either via a component, tag or custom predicate

Reply:
```suggestion
!!! warning "Nest Equipability"
    Make sure to remember the equipability of your nest. If your Nested Accessory needs to change its equipability dynamically, you might want to add the Slot Validation component to the nest. Alternatively, just using a tag or Tag Predicate is fine.
```